### ISSUE TITLE
Support for Linux AARCH64 

### ISSUE THREAD

**Issue by raydac** [jinput/jinput#181]
Do you have plans to provide support Linux+AARCH64?  For instance Linux Raspberry Pi 4 throws such error
![image](https://github.com/jinput/jinput/assets/3008810/4801235c-8bb1-4acc-83d8-0cccc8cd403c)

**Comment by Valkryst** [comment]
I'm sure it's possible to add support for it, though I'm unsure if Endolf would do it or has the time to do it. Maybe he'll be available to review and merge a PR, which add support for Linux+AARCH64, if you or someone else has the time to add it 🤔

**Comment by Endolf** [comment]
Yup, if someone wants to add a new build target and submit a PR (don't forget the github actions) then I would be happy to review it, I don't think that adding new build targets makes sense unless we are doing proper release.

**Comment by kittylyst** [comment]
See PR #310
